website/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "rslib-website",
+  "version": "0.0.1",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "rspress build",
+    "dev": "rspress dev",
+    "preview": "rspress preview"
+  },
+  "devDependencies": {
+    "@rsbuild/core": "1.0.1-rc.5",
```

Comment:
@rsbuild/core should not be here?

Reply:
use `/// <reference types="@rsbuild/core/types" />` in `env.d.ts` to declare assets module